_translations/en.yml

```diff
@@ -350,7 +350,7 @@
     howitworkstxt1: "From a user perspective, Bitcoin is nothing more than a mobile app or computer program that provides a personal Bitcoin wallet and allows a user to send and receive bitcoins with them. This is how Bitcoin works for most users."
     howitworkstxt2: "Behind the scenes, the Bitcoin network is sharing a public ledger called the \"block chain\". This ledger contains every transaction ever processed, allowing a user's computer to verify the validity of each transaction. The authenticity of each transaction is protected by digital signatures corresponding to the sending addresses, allowing all users to have full control over sending bitcoins from their own Bitcoin addresses. In addition, anyone can process transactions using the computing power of specialized hardware and earn a reward in bitcoins for this service. This is often called \"mining\". To learn more about Bitcoin, you can consult the <a href=\"#how-it-works#\">dedicated page</a> and the <a href=\"#bitcoin-paper#\">original paper</a>."
     used: "Is Bitcoin really used by people?"
-    usedtxt1: "Yes. There is a <a href=\"http://usebitcoins.info/\">growing number of businesses</a> and individuals using Bitcoin. This includes brick and mortar businesses like restaurants, apartments, law firms, and popular online services such as Namecheap, WordPress, and Reddit. While Bitcoin remains a relatively new phenomenon, it is growing fast. At the end of August 2013, the <a href=\"https://bitcoincharts.com/bitcoin/\">value of all bitcoins in circulation</a> exceeded US$ 1.5 billion with millions of dollars worth of bitcoins exchanged daily."
+    usedtxt1: "Yes. There is a <a href=\"/en/spend-bitcoin\">growing number of businesses</a> and individuals using Bitcoin. This includes brick-and-mortar businesses like restaurants, apartments, and law firms, as well as popular online services such as Namecheap, Overstock.com, and Reddit. While Bitcoin remains a relatively new phenomenon, it is growing fast. At the end of April 2017, the <a href=\"https://bitcoincharts.com/bitcoin/\">total value of all existing bitcoins</a> exceeded 20-billion US dollars, with millions of dollars worth of bitcoins exchanged daily."
```

Comment:
Please change "Yes, There is a" to "Yes, There are a" (I know it was already like that)

Also, please change "20-billion" to "20 billion" (no hyphen)